main.go

```diff
@@ -52,6 +52,7 @@ func main() {
 	subcommands.Register(&cmd.SchemaCmd{}, "")
 	subcommands.Register(&cmd.DataCmd{}, "")
 	subcommands.Register(&cmd.SchemaAndDataCmd{}, "")
+	subcommands.Register(&cmd.CleanupCmd{}, "")
```

Comment:
Is it possible that the cleanup be a subcommand of the flow type?

Reply:
Do you mean forward and reverse flow? 
Yes, ideally, all the 4 commands above should be context of the flow type...we only have one today though so I kept it at same level. 
Maybe we refactor all of these together once we introduce the concept of a "flow"? I don't want this to become a confusing experience until we have that..